packages/@okta/vuepress-site/docs/guides/email-magic-links-overview/main/index.md

```diff
@@ -177,10 +179,9 @@ To point the magic link directly to the endpoint in your application, you must r
 
 1. In the **Admin Console**, go to **Customizations** > **Emails**.
 2. On the **Emails** page, find the **Other** category on the template menu.
-3. Under **Other**, click **Email Challenge**.
-4. On the **Email Challenge** email template page, click **Edit**.
-5. Under **Default Email**, click **Edit**.
-6. In the Message field, locate the magic link in the field's HTML. The link is located in the href attribute of an `<a>` tag with the `id` of `email-authentication-button`. It looks like this:
+3. Under **Other**, click **Email Challenge**, and then on the **Email Challenge** email template page, click **Edit**.
+4. Under **Default Email**, click **Edit**.
```

Comment:
```suggestion
4. Select **Default Email** > **Edit**.
```

Reply:
^ you don't have to keep this edit if you don't agree. I'm just not a fan of "under" in procedure steps.

Reply:
they aren't selecting **Default Email** - I'll change it to in the Default Email section